config.json

```diff
@@ -0,0 +1,9 @@
+
+{
+	"apiKey": "AIzaSyDjS6tGpqjAaC0aXQAGZMkjssABbZzTgSo",
```

Comment:
Ouch! can you put this in a gitignore file please? This should not be public!

Reply:
Yes, this means the API key is compromised and should be regenerated. Once on the internet, always on the internet.

Reply:
This config is already public and is part of the Gatsby & Firebase starter that I've been testing out: https://github.com/muhajirdev/gatsby-starter-firebase/blob/master/src/firebase.js 

But yes, API keys shouldn't be public.